src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -81,8 +84,8 @@ public DefaultClient(URL baseUrl,  OkHttpClient client, IResourceFactory factory
 		if(this.factory != null) {
 			this.factory.setClient(this);
 		}
-		openShiftVersion = System.getProperty(SYSTEM_PROP_OPENSHIFT_API_VERSION, null);
-		kubernetesVersion = System.getProperty(SYSTEM_PROP_K8E_API_VERSION, null);
+		initMasterVersion("version/openshift", new VersionCallback(version -> this.openShiftVersion = version));
```

Comment:
Before changing these, please ensure they are not used in some other capacity. There original intention represented API version of the endpoints and not version of the overall product.